addons/api/addon/models/worker.js

```diff
@@ -51,10 +50,35 @@ export default class WorkerModel extends GeneratedWorkerModel {
     }
 
     return Object.entries(this.config_tags).flatMap(([key, value]) =>
-      value.map((tag) => ({ key, value: tag })),
+      value.map((tag) => ({ key, value: tag, type: 'config' })),
+    );
+  }
+
+  /**
+   * Returns the api tags as an array of key/value pair objects.
+   * @type {[object]}
+   */
+  getApiTagList() {
```

Comment:
Unrelated to your PR but looking at this I'm pretty sure I should have made `getConfigTagList` into a getter instead of a function lol

Reply:
Well I can fix that now, no worries